### DISCUSSION TITLE
Janet 1.35.2

### DISCUSSION THREAD

**Discussion by bakpakin** [janet-lang/janet#1462] [General]
Patch release to fix some issues with 1.35.0

<hr /><em>This discussion was created from the release <a href='https://github.com/janet-lang/janet/releases/tag/v1.35.2'>Janet 1.35.2</a>.</em>

**Comment by sogaiu** [comment]
`import` with `:only` without quotes works nicely :)

```
$ janet
Janet 1.35.2-fda0a081 linux/x64/gcc - '(doc)' for help
repl:1:> (import spork/fmt :only [format])
@{_ @{:value <cycle 0>} fmt/format @{:private true}}
```

Thanks!
